freesms/__init__.py

```diff
@@ -78,5 +79,12 @@ def send_sms(self, text, **kw):
             # remove SSL warning
             requests.packages.urllib3.disable_warnings(InsecureRequestWarning)
 
-        res = requests.get(FreeClient.BASE_URL, params=params, **kw)
-        return FreeResponse(res.status_code)
+        try:
+            res = requests.get(FreeClient.BASE_URL, params=params, **kw)
+            return FreeResponse(res.status_code)
+        except ConnectionError:
+            return FreeResponse(444)
+        except Timeout:
+            return FreeResponse(499)
+        except RequestException:
+            return FreeResponse(444)
```

Comment:
I think this whole `FreeResponse` class is a design mistake: it doesn’t really serve anything. `FreeClient#send_sms` shoud return the `requests.Response` it gets, and maybe wrap any exception in its own class. Something like this:

```python
try:
    res = requests.get(FreeClient.BASE_URL, params=params, **kw)
    res.raise_for_status()
except RequestException as e:  # all the other exceptions derive from that one anyway
    raise FreeException(e)
```